### PR TITLE
Fix dex quotes

### PR DESCRIPTION
**Pull request by Pantani**
```
curl -X POST \
  http://localhost:8420/v1/market/ticker \
  -d '{
  "currency": "USD",
  "assets": [
    {
      "coin": 714,
      "type": "token",
      "token_id": "RAVEN-F66"
    }
  ]
}'
{"currency":"USD","docs":[{"coin":714,"token_id":"RAVEN-F66","type":"token","price":{"value":0.00016121268115623202,"change_24h":0.09},"last_update":"2019-11-27T22:11:47.494894-03:00"}]}
```

```
curl -X POST \
  http://localhost:8420/v1/market/ticker \                
  -d '{             
  "currency": "USD",
  "assets": [
    {
      "coin": 714,
      "type": "token",
      "token_id": "BUSD-BD1"
    }
  ]
}'
{"currency":"USD","docs":[{"coin":714,"token_id":"BUSD-BD1","type":"token","price":{"value":0.9907856572387254,"change_24h":3.86},"last_update":"2019-11-27T22:08:45.826068-03:00"}]}
```

```
curl -X POST \
  http://localhost:8420/v1/market/ticker \
  -d '{
  "currency": "USD",
  "assets": [
    {
      "coin": 714,
      "type": "coin"
    },
    {
      "coin": 714,
      "type": "token",
      "token_id": "BUSD-BD1"
    }
  ]
}'
{"currency":"USD","docs":[{"coin":714,"type":"coin","price":{"value":16.0990958111,"change_24h":2.79745},"last_update":"2019-11-28T02:23:07Z"},{"coin":714,"token_id":"BUSD-BD1","type":"token","price":{"value":1.008721596695468,"change_24h":3.3},"last_update":"2019-11-27T23:24:13.733085-03:00"}]}
```

### CODE REVIEW

File: marketdata/market/dex/dex.go [hunk 1]
```diff
@@ -59,9 +59,9 @@ func normalizeTicker(price *CoinPrice, provider string) (*blockatlas.Ticker, err
 		return nil, errors.E(err, "normalizeTicker parse value24h error",
 			errors.Params{"PriceChange": price.PriceChangePercent, "Symbol": price.BaseAssetName})
 	}
-	tokenId := price.QuoteAssetName
+	tokenId := price.BaseAssetName
 	if tokenId == BNBAsset {
-		tokenId = price.BaseAssetName
+		tokenId = price.QuoteAssetName
```
Comment (vikmeup) [inline marketdata/market/dex/dex.go]: is there a way to add unit test for this?

File: config.yml [hunk 2]
```diff
@@ -33,13 +33,13 @@ observer:
 market:
   enabled: false
   dex:
-    quote_update_time: 2m
+    quote_update_time: 20m
     api: https://dex.binance.org/api
   cmc:
```
Comment (vikmeup) [inline config.yml]: 👍

File: marketdata/market/dex/dex.go [hunk 3]
```diff
@@ -40,7 +40,7 @@ func (m *Market) GetData() (blockatlas.Tickers, error) {
 		return nil, errors.E(err, "rate not found", errors.Params{"asset": BNBAsset})
 	}
 	result := normalizeTickers(prices, m.GetId())
-	result.ApplyRate(rate.Rate, blockatlas.DefaultCurrency)
+	result.ApplyRate(1/rate.Rate, blockatlas.DefaultCurrency)
```
Comment (vikmeup) [inline marketdata/market/dex/dex.go]: can maybe this one too for unit test

File: marketdata/market/dex/dex.go (hunk 1 above)
Comment (Pantani) [reply marketdata/market/dex/dex.go]: we already have, I need to fix that

File: marketdata/market/dex/dex.go (hunk 3 above)
Comment (Pantani) [reply marketdata/market/dex/dex.go]: yes

**Pull request merged**